apps/web/components/Organization/Slice/OverviewLinks/OverviewLinks.tsx

```diff
@@ -76,16 +77,17 @@ export const OverviewLinksSlice: React.FC<SliceProps> = ({ slice }) => {
                               )}{' '}
                             </Box>
                           )}
-                          <Link
+                          <a
```

Comment:
Why the need for switching to an `a` tag instead of Link?
Are these internal island.is links that are supposed to open as a new window?

If not then they should open up externally automatically from the else statement of the `isInternal` block:
https://github.com/island-is/island.is/blob/71eb2514ef3bb2d012d0050e852ae2dd42dd0e65/libs/island-ui/core/src/lib/Link/Link.tsx#L88-L92

Reply:
Yes, we got a request from Digital Iceland to make them open in a new tab. The `Link` component does not take a `target` property so I thought this was the simplest method.

Reply:
:+1:

Reply:
Could, and should, we create a prop on the Link component `libs/island-ui/core/src/lib/Link/Link.tsx` called `newTab`?

Then in the relevant lines for internal:
https://github.com/island-is/island.is/blob/71eb2514ef3bb2d012d0050e852ae2dd42dd0e65/libs/island-ui/core/src/lib/Link/Link.tsx#L78-L82

Adding:
```diff
            className={classNames}
            {...linkProps}
+           {...(newTab && {target: ‘_blank’})}
            tabIndex={skipTab ? -1 : undefined}
```

Reply:
:+1: